users/models.py

```diff
@@ -51,6 +51,7 @@ class User(TimeStampedMixin, UUIDMixin, AbstractUser):
         default=RolesTypes.USER,
         max_length=50,
     )
+    avatar = models.ImageField(upload_to='images/', blank=True)
```

Comment:
Давай в отдельную директорию их складывать будем. Пока не знаю зачем, но хуже не будет.
```suggestion
    avatar = models.ImageField(upload_to='avatars/', blank=True)
```